packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -35,8 +35,57 @@ class PlaceOrderError extends Error {
     }
 }
 
+class DefaultGetCheckoutError extends Error {
+    constructor (message, errorCode) {
+        super(message);
+        this.messageKey = 'errorMessages.pageLoad.description';
+        this.eventToEmit = EventNames.CheckoutGetFailure;
+        this.logMessage = 'Get Checkout Failure';
+        this.errorCode = errorCode;
+        this.shouldShowInDialog = false;
+        this.errorFormType = 'pageLoad';
+    }
+}
+
+class AccessForbiddenError extends DefaultGetCheckoutError {
+    constructor (message) {
+        super(message);
+        this.messageKey = 'errorMessages.accessForbiddenError.description';
+        this.logMessage = 'Get Checkout Failure: Access Forbidden';
+        this.errorFormType = 'accessForbiddenError';
+    }
+}
+
+class AvailableFulfilmentGetError extends Error {
+    constructor (message, errorCode) {
+        super(message);
+        this.messageKey = 'errorMessages.pageLoad.description';
+        this.eventToEmit = EventNames.CheckoutAvailableFulfilmentGetFailure;
+        this.logMessage = 'Get Checkout Available Fulfilment Times Failure';
+        this.errorCode = errorCode;
+        this.shouldShowInDialog = false;
+        this.errorFormType = 'pageLoad';
+    }
+}
+
+class GetCheckoutLoadBasketError extends Error {
```

Comment:
I'd rename this to `GetBasketError`, since technically it doesn't have anything to do with the checkout bits :)